bitwarden_license/bit-web/src/app/sm/secrets/secrets.component.ts

```diff
@@ -8,18 +10,29 @@ import { SecretApiService } from "./secret-api.service";
   selector: "sm-secrets",
   templateUrl: "./secrets.component.html",
 })
-export class SecretsComponent implements OnInit {
+export class SecretsComponent implements OnInit, OnDestroy {
   private organizationId: string;
+  private destroy$: Subject<void> = new Subject<void>();
```

Comment:
We don't need explicit type definitions if it's equivalent to the right side.

```suggestion
  private destroy$ = new Subject<void>();
```